content/articles/picking-pdf-and-image-from-phone-storage/index.md

```diff
@@ -0,0 +1,322 @@
+                }
+            }
+        })
+        myAlertDialog.show()
+    }
+
+    private fun selectPdf() {
+        val pdfIntent = Intent(Intent.ACTION_GET_CONTENT)
+        pdfIntent.type = "application/pdf"
+        pdfIntent.addCategory(Intent.CATEGORY_OPENABLE)
+        startActivityForResult(pdfIntent, 12)
+    }
+
+    @SuppressLint("Range")
+   override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+
+        // For loading Image
+        if (resultCode != RESULT_CANCELED) {
+            when (requestCode) {
+                0 -> if (resultCode == RESULT_OK && data != null) {
+                    val imageSelected = data.extras!!["data"] as Bitmap?
+                    imageView.setImageBitmap(imageSelected)
+                }
+                1 -> if (resultCode == RESULT_OK && data != null) {
+                    val imageSelected = data.data
+                    val pathColumn = arrayOf(MediaStore.Images.Media.DATA)
+                    if (imageSelected != null) {
+                        val myCursor = contentResolver.query(
+                            imageSelected,
+                            pathColumn, null, null, null
+                        )
+                        if (myCursor != null) {
+                            myCursor.moveToFirst()
+                            val columnIndex = myCursor.getColumnIndex(pathColumn[0])
+                            val picturePath = myCursor.getString(columnIndex)
+                            imageView.setImageBitmap(BitmapFactory.decodeFile(picturePath))
+                            myCursor.close()
+                        }
+                    }
+                }
+            }
+        }
+
+        // For loading PDF
+        when (requestCode) {
+            12 -> if (resultCode == RESULT_OK) {
+
+                pdfUri = data?.data!!
+                val uri: Uri = data?.data!!
+                val uriString: String = uri.toString()
+                var pdfName: String? = null
+                if (uriString.startsWith("content://")) {
+                    var myCursor: Cursor? = null
+                    try {
+                        myCursor = applicationContext!!.contentResolver.query(uri, null, null, null, null)
+                        if (myCursor != null && myCursor.moveToFirst()) {
+                            pdfName = myCursor.getString(myCursor.getColumnIndex(OpenableColumns.DISPLAY_NAME))
+                            pdfTextView.text = pdfName
+                        }
+                    } finally {
+                        myCursor?.close()
+                    }
+                }
+            }
+        }
+    }
+```
+[Screenshot](!/engineering-education/picking-pdf-and-image-from-phone-storage/screenshot.png)
+
+### Coclusion
+This tutorial is only meant to guide you through. You need to try the codes provided to practice and master the concept. 
```

Comment:
Change to " This tutorial is only meant to guide you through. First, you need to try the codes provided to practice and master the concept. "